CONTRIBUTING.md

```diff
@@ -0,0 +1,85 @@
+# How to contribute ?
+
+This is a guide intended for developers wanting to lend a hand.
+It contains detailed instructions on how to setup your development environment and
+the prerequisites needed to work on this project.
+## Prerequisites
+
+Make sure you have R installed. On macOs:
+
+```bash
+# libgit2 is required by devtools
+brew install r libgit2
+brew install --cask rstudio  # recommended
+
+R --version  # make sure you have 4.1.2, the latest as of writing
+```
+
+If you have never created an R package before, it is good idea to follow the tutorial here: https://r-pkgs.org/whole-game.html
+
+## Installing deps
+
+```bash
+make install
+```
+
+Make sure things are properly installed>
+
+```text
+> library(devtools)
+Loading required package: usethis
+> has_devel()
+Your system is ready to build packages!
+```
+
+## Quickstart
+
+Install with:
+
+```R
+# type="source" prevents a warning about R version
+install.packages("...", repos=NULL, type="source")
```

Comment:
Replace with Github release URL?

Reply:
Well that's for installation from source, if we are developing, we don't necessarily want to take the released version ?